client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -65,6 +71,35 @@ const CancelJetpackForm: React.FC< Props > = ( {
 		isSiteAutomatedTransfer( state, purchase.siteId )
 	);
 
+	const cancellationOffer = useSelector( ( state ) => {
+		const offers = getCancellationOffers( state, purchase.id );
+		if ( offers.length > 0 ) {
+			return offers[ 0 ];
+		}
+
+		return null;
+	} );
+
+	const fetchingCancellationOffers = useSelector( ( state ) =>
+		isFetchingCancellationOffers( state )
+	);
+
+	const isOfferPriceSameOrLowerThanPurchasePrice = useMemo( () => {
+		return cancellationOffer ? purchase.amount >= cancellationOffer.originalPrice : false;
+	}, [ cancellationOffer, purchase ] );
```

Comment:
We don't need to memoize whole objects if we only read `amount` and `originalPrice` values

```suggestion
	}, [ cancellationOffer.originalPrice, purchase.amount ] );
```

Reply:
I think since we check whether `cancellationOffer` is non-falsy first, we should have the full object as a dependency. (The linter prefers this as well.) Good call on just passing `purchase.amount`.